src/interface/src/app/plan/plan-summary/saved-scenarios/saved-scenarios.component.html

```diff
@@ -28,6 +28,11 @@
           ? 'Click “New scenario” to start creating your scenarios.'
           : ''
       ">
+      <mat-slide-toggle
+        [(ngModel)]="showOnlyMyScenarios"
+        (change)="fetchScenarios()">
+        Show only my scenarios
+      </mat-slide-toggle>
```

Comment:
Our nice UI design has this nested inside what would be the mat-tab-header, which doesn't seem possible without a hacky solution, so I'm looking for alternatives.